ds-caselaw-ingester/tests.py

```diff
@@ -521,8 +521,8 @@ def test_get_consignment_reference_empty_v2(self):
         message["parameters"][
             "bundleFileURI"
         ] = "http://172.17.0.2:4566/te-editorial-out-int/ewca_civ_2021_1881.tar.gz"
-        result = lambda_function.get_consignment_reference(message)
-        assert result == "ewca_civ_2021_1881"
+        with pytest.raises(lambda_function.InvalidMessageException):
```

Comment:
Parsing the consignment reference from the filename was always a hack; we have changed the assumption to "the consignment reference always exists"